mlir/test/Conversion/ArithToEmitC/arith-to-emitc.mlir

```diff
@@ -390,3 +390,94 @@ func.func @arith_int_to_float_cast_ops(%arg0: i8, %arg1: i64) {
 
   return
 }
+
+// -----
+
+func.func @arith_cmpi_eq(%arg0: i32, %arg1: i32) -> i1 {
```

Comment:
The cmpi test here and below seem not to belong here?

Reply:
I forgot to drop them from the test file :)